src/app/cli/src/tests/coda_shared_prefix_multiproducer_test.ml

```diff
@@ -3,22 +3,36 @@ open Async
 
 let name = "coda-shared-prefix-multiproducer-test"
 
+let runtime_config =
+  lazy
+    ( (* TODO: Decide on a profile for this test.
+         (This uses multiple profiles on CI)
+      *)
+      "{}" |> Yojson.Safe.from_string |> Runtime_config.of_yojson
```

Comment:
It's not safe to land this as is right? We could make some input to switch between the different profiles right?

Reply:
This will fall through to the compiled configuration, so it should still work as intended! This effectively works as an empty config file.